helm/templates/deployment.yaml

```diff
@@ -134,21 +134,17 @@ spec:
           value: {{ $.Values.minio.metricsLogger.REDIS_HOST | quote }}
         - name: REDIS_PORT
           value: {{ $.Values.minio.metricsLogger.REDIS_PORT | quote }}
-        - name: REDIS_POOL_MAX_CONNECTIONS
-          value: {{ $.Values.minio.metricsLogger.REDIS_POOL_MAX_CONNECTIONS | quote }}
-        - name: REDIS_POOL_TIMEOUT
-          value: {{ $.Values.minio.metricsLogger.REDIS_POOL_TIMEOUT | quote }}
-        - name: REDIS_KEY_PREFIX_DEPLOYMENT_LAST_ACTION
-          value: {{ $.Values.minio.metricsLogger.REDIS_KEY_PREFIX_DEPLOYMENT_LAST_ACTION | quote }}
         - name: UPDATE_GRACE_PERIOD_SECONDS
           value: {{ $.Values.minio.metricsLogger.UPDATE_GRACE_PERIOD_SECONDS | quote }}
         - name: DEPLOYMENT_API_METRICS_FLUSH_INTERVAL_SECONDS
           value: {{ $.Values.minio.metricsLogger.DEPLOYMENT_API_METRICS_FLUSH_INTERVAL_SECONDS | quote }}
+        - name: REDIS_KEY_PREFIX_DEPLOYMENT_LAST_ACTION
+          value: {{ $.Values.minio.metricsLogger.REDIS_KEY_PREFIX_DEPLOYMENT_LAST_ACTION | quote }}
         - name: REDIS_KEY_PREFIX_DEPLOYMENT_API_METRIC
           value: {{ $.Values.minio.metricsLogger.REDIS_KEY_PREFIX_DEPLOYMENT_API_METRIC | quote }}
-        {{ if $.Values.minio.metricsLogger.debug }}
-        - name: DEBUG
-          value: "yes"
+        {{ if $.Values.minio.metricsLogger.LOG_LEVEL }}
+        - name: LOG_LEVEL
+          value: "debug"
         {{ end }}
```

Comment:
Please add all the possible env vars for configuration of the providers here, otherwise there is no way to configure them from the helm values